moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -136,8 +136,12 @@ enum IkParameterizationType
   /// when serializing the ik parameterizations
 };
 
+// HACK to fix isnan ambiguity
+#undef isnan
```

Comment:
isnan could be defined somewhere else and I thought redefining macros would throw an error but it's only a warning so maybe we want to get the warning.